main/inc/lib/extra_field.lib.php

```diff
@@ -2468,79 +2468,81 @@ public function get_where_clause($col, $oper, $val)
     public function getExtraFieldRules($filters, $stringToSearch = 'extra_')
     {
         $extra_fields = [];
+        $condition_array = [];
 
         // Getting double select if exists
         $double_select = [];
-        foreach ($filters->rules as $rule) {
-            if (empty($rule)) {
-                continue;
-            }
-            if (strpos($rule->field, '_second') === false) {
-            } else {
-                $my_field = str_replace('_second', '', $rule->field);
-                $double_select[$my_field] = $rule->data;
+        if (is_object($filters) && property_exists($filters, 'rules') && is_array($filters->rules) && !empty($filters->rules)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 128 characters](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4663904)